test/tests/variables/block_aux_kernel/tests

```diff
@@ -1,8 +1,13 @@
 [Tests]
+  design = 'syntax/AuxVariables/index.md'
+  issues = ''
```

Comment:
Odd - from here on down you supplied no issue. I thought MooseDocs didn't allow this anymore?